ensime_shared/protocol.py

```diff
@@ -94,6 +97,12 @@ def handle_type_inspect(self, call_id, payload):
     def show_type(self, call_id, payload):
         raise NotImplementedError()
 
+    def handle_source_positions(self, call_id, payload):
+        raise NotImplementedError()
```

Comment:
Should we open an issue so that contributors fill this in at some point of the future?